app/src/main/java/com/gdsctsec/smartt/ui/weekday/WeekdayActivity.kt

```diff
@@ -23,10 +23,10 @@ class WeekdayActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_weekday)
 
-        val imageView: ImageView = findViewById(R.id.empty_list_image_view)
+        val imageViewCalendarImageWhenEmpty: ImageView = findViewById(R.id.empty_list_image_view)
         val lecturesRecyclerView: RecyclerView = findViewById(R.id.lecture_recycler_view)
-        val lecNumber: TextView = findViewById(R.id.lecture_number_text_view)
-        val timeList: FloatingActionButton = findViewById(R.id.lecture_add_floating_action_button)
+        val lecNumberCountTextView: TextView = findViewById(R.id.lecture_number_text_view)
+        val addNewLectureEventFloatingActionButton: FloatingActionButton = findViewById(R.id.lecture_add_floating_action_button)
 
         //recycler View Adapter
         val timeListRecyclerView: List<String> = listOf(
```

Comment:
you forgot these two!
rename it to `timeList` and the below to `subjectList` because they are of type List